omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -73,8 +73,10 @@
     <Property Id="MSIRESTARTMANAGERCONTROL" Value="Disable" />
 
     <Property Id="DDAGENTUSER_NAME" />
-    <Property Id="DDAGENTUSER_PASSWORD"  />
-
+    <Property Id="DDAGENTUSER_PASSWORD" Hidden="yes" />
```

Comment:
For what reason was  `DDAGENTUSER_PASSWORD` already a property but `APIKEY` wasn't? What's the effect of something being a property here? Both can be passed in as parameters so it must be something else?